symfony/plugins/orangehrmPerformancePlugin/test/Dao/PerformanceReviewDaoTest.php

```diff
@@ -11,17 +11,27 @@
  * @author nadeera
  */
 
+namespace OrangeHRM\Performance\test\Dao;
+
+use OrangeHRM\Performance\Dao\PerformanceReviewDao;
+use PerformanceReview;
+use PHPUnit_Framework_TestCase;
+use sfConfig;
+use TestDataService;
+
 /**
  * @group performance
  */
-class PerforamanceReviewDaoTest extends PHPUnit_Framework_TestCase {
+class PerforamanceReviewDaoTest extends TestCase {
```

Comment:
Typo. You will have to change the file name also
```suggestion
class PerformanceReviewDaoTest extends TestCase {
```

Reply:
changed